Postfix/postfix/ingest/parser.yml

```diff
@@ -353,7 +353,11 @@ stages:
 
       - set:
           network.protocol: "smtp"
-        filter: "{{event.message.POSTFIX_SMTPD_CONNECT != None and event.message.tmp_ESMTP != None}}"
+        filter: "{{event.message.POSTFIX_SMTPD_CONNECT != null and event.message.tmp_ESMTP != null}}"
+
+      - set:
+          network.protocol: "smtp"
+        filter: "{{original.get('log', {}).get('syslog', {}).get('appname', '') == 'postfix'}}"
```

Comment:
I feel like the syntax here could be simpler like

original.log.syslog.appname == postfix

Reply:
You're right. I thought there was en error with this, but I rechecked and it seems that I confused this with another issue